src/components/Box/Box.cy.tsx

```diff
@@ -0,0 +1,35 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { FC } from 'react';
+import { Box } from './Box';
+
+const Component: FC<JSX.IntrinsicElements['div']> = (props) => {
```

Comment:
Is it really necessary to create a new component here as you mount the same in every test and anyway you can pass whatever to the Box component.